src/components/ToggleSwitch/index.tsx

```diff
@@ -115,13 +116,13 @@ const ToggleSwitch: React.FC<ToggleSwitchProps> = ({
   onValueChange,
   labelLeft,
   labelRight,
-  circleActiveColor = '#00ADB5',
+  circleActiveColor = COLORS.holytosca,
   circleInActiveColor = colors.blue1,
-  backgroundActive = '#F5F5F5',
-  backgroundInactive = '#F5F5F5',
+  backgroundActive = COLORS.lightgrey,
+  backgroundInactive = COLORS.lightgrey,
   styleLabelLeft,
   styleLabelRight,
-  activeTextColor = '#00ADB5',
+  activeTextColor = COLORS.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These changes replace hard-coded color values with dynamic ones from a theme object. This is a good practice as it centralizes color definitions and makes it easier to manage and update colors across the application.
```diff
-  circleActiveColor = '#00ADB5',
+  circleActiveColor = COLORS.holytosca,
-  backgroundActive = '#F5F5F5',
-  backgroundInactive = '#F5F5F5',
+  backgroundActive = COLORS.lightgrey,
+  backgroundInactive = COLORS.lightgrey,
-  activeTextColor = '#00ADB5',
+  activeTextColor = COLORS.holytosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->